airflow-core/src/airflow/ui/src/i18n/locales/he/common.json

```diff
@@ -0,0 +1,79 @@
+{
+  "defaultToGraphView": "ברירת מחדל תצוגת גרף",
+  "defaultToGridView": "ברירת מחדל תצוגת רשת",
+  "logout": "התנתק",
+  "switchToDarkMode": "החלף למצב כהה",
+  "switchToLightMode": "החלף למצב בהיר",
+  "timezone": "איזור זמן",
+  "user": "משתמש",
+  "language": {
+    "chinese": "סינית מסורתית",
+    "english": "אנגלית",
+    "german": "גרמנית",
+    "korean": "קוריאנית",
+    "select": "בחר"
+  },
+  "nav": {
+    "home": "מסך הבית",
+    "assets": "משאבים (Assets)",
```

Comment:
משאבים זה "ריסורסז". לדעתי עדיף תוצאים\תוצרים (תרגום של artifact).

Reply:
changed to נכסים